dashboard/views_test.py

```diff
@@ -38,6 +39,12 @@ def setUpTestData(cls):
             uid="{}_edx".format(cls.user.username),
             extra_data={"access_token": "fooooootoken"}
         )
+        cls.user_access_cache = UserCacheRefreshTimeFactory(
```

Comment:
we do not user `cls.user_access_cache`, but the entry in `UserCacheRefreshTime` is necessary (note that is set in the future).
Removing the class variable anyway